doc/connectivity/connectome_extraction.rst

```diff
@@ -0,0 +1,232 @@
+.. _connectome_extraction:
+
+============================================================================
+Connectome extraction: inverse covariance for direct connections 
+============================================================================
+
+.. topic:: **Page summary**
+
+   Given a set of time-series (eg as extracted in the previous section)
+   A *functional connectome* is a set of connections representing brain
```

Comment:
"brain interactions" appears to be an unusual phrasing to me.
Alternative:
- functional coupling
- mutual influence
- reciprocal communication/interaction